src/common/options/mds.yaml.in

```diff
@@ -206,6 +206,14 @@ options:
   services:
   - mds
   with_legacy: true
+- name: mds_heartbeat_reset_grace
```

Comment:
Just a Q: Is this heartbeat also used to check whether node is active or not, like in real :)

Reply:
The heartbeat here is just for the MDS daemon's liveness. And the MDS should periodically tell the Monitors that it's alive or not stuck.

Reply:
Is this grace a **time period** or **number of iterations** ?
Does `mds_heartbeat_reset_grace_period` sound correct or `mds_heartbeat_reset_grace_iterations` sound correct ?